packages/kbn-coloring/src/shared_components/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { withSuspense } from '@kbn/shared-ux-utility';
+
+/**
+ * The Lazily-loaded `CustomizablePalette` component.  Consumers should use `React.Suspense` or
+ * the withSuspense` HOC to load this component.
+ */
+export const CustomizablePaletteLazy = React.lazy(() => import('./coloring'));
```

Comment:
@spalger Just to make sure - does lazy loading within a package actually work? Or will it end up in a single chunk and the lazy-loading should be moved into the consuming plugin?

Reply:
My bad, got confused. It works as intended from what I can tell. Please disregard